### DISCUSSION TITLE
error: Could not find a video on this page

### DISCUSSION THREAD

**Discussion by wtfismyname69420** [streamlink/streamlink#3878] [Q&A] [answered]
Hello, I got this error and I don't know what to do about it.

ubuntu@ubuntu:~$ streamlink -l debug https://www.youtube.com/channel/UCeY0bbntWzzVIaj2z3QigXg/live best
[cli][debug] OS:         Linux-5.4.0-1041-raspi-armv7l-with-glibc2.29
[cli][debug] Python:     3.8.10
[cli][debug] Streamlink: 1.7.0
[cli][debug] Requests(2.22.0), Socks(1.6.7), Websocket(0.53.0)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/channel/UCeY0bbntWzzVIaj2z3QigXg/live
error: Could not find a video on this page

I'm running this on my raspberry pi 3 and I think that  might be the issue here but I'm not sure and I want to find out what this issue may be,

**Comment by bastimeyer** [comment, marked as answer]
> Streamlink: 1.7.0

https://streamlink.github.io/changelog.html
https://streamlink.github.io/install.html
